extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json

```diff
@@ -226,37 +226,37 @@
           "type": "object",
           "description": "AJO Orchestrated Campaign Entity Specific Fields",
           "properties": {
-            "https://ns.adobe.com/experience/customerJourneyManagement/entities/orchestratedCampaign/campaignVersionID": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/campaignVersionID": {
               "title": "Orchestrated Campaign Version ID",
               "type": "string",
               "description": "Represents a frozen version of a Orchestrated Campaign"
             },
-            "https://ns.adobe.com/experience/customerJourneyManagement/entities/orchestratedCampaign/name": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/campaignName": {
```

Comment:
Is this an intentional change, we are changing name to campaignName? This will be inconsistent with "name" field defined in single step campaign.